src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -7,12 +7,18 @@
 .form-check-inline {
     .ibexa-input {
         position: relative;
+
+        &--checkbox {
+            margin-bottom: calculateRem(1px);
+        }
+
+        &--radio {
+            margin-bottom: 0;
```

Comment:
Is it needed?

Reply:
Yes, with:
![Screenshot 2021-07-14 at 12 06 43](https://user-images.githubusercontent.com/10233057/125604550-a9aa7e5c-3a31-45b6-94b7-18059fab87d9.png)
and without:
![Screenshot 2021-07-14 at 12 06 00](https://user-images.githubusercontent.com/10233057/125604521-2433f1f0-092c-41f6-ac8f-3cfb411cf34e.png)
Without it checkbox is not centered properly.

Reply:
In the case of "normal" radio-button we can align radio with label using label margings. Here label takes all available space.